src/components/MessageBar/index.tsx

```diff
@@ -0,0 +1,44 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import { NestableBaseComponentProps } from '../../util/BaseComponent/props';
+import { Enum } from 'typescript-string-enums';
+import classNames = require('classnames');
+import './message-bar.css';
+
+export const MessageBarType = Enum({
+  INFO: 'info',
+  WARNING: 'warning',
+  ERROR: 'error',
+});
+export type MessageBarType = Enum<typeof MessageBarType>;
+
+export interface MessageBarProps extends NestableBaseComponentProps {
+  ariaLabel: string;
+  type?: MessageBarType;
+  actions?: JSX.Element;
+  children: React.ReactNode[] | string;
```

Comment:
`React.ReactNode` includes `string`... Have you tried just doing `children: React.ReactNode`, without the array?

Reply:
i just did what you said before.... i can change it to that